redirects/kmm.yml

```diff
@@ -0,0 +1,66 @@
+- from:
+   - /docs/mobile/home.html
```

Comment:
index pages like ‘/docs/mobile’ required roo.

Reply:
I've added `/docs/mobile`
should I add `/docs/mobile/` too (with a slash sign)?